ckanext/validation/plugin.py

```diff
@@ -316,3 +321,43 @@ def _run_async_validation(resource_id):
         log.warning(
             u'Could not run validation for resource {}: {}'.format(
                 resource_id, str(e)))
+
+
+def _should_remove_unsupported_resource_validation_reports(res_dict):
+    if not t.h.asbool(t.config.get('ckanext.validation.limit_validation_reports', False)):
```

Comment:
Remember to document

```suggestion
    if not t.h.asbool(t.config.get('ckanext.validation.clean_validation_reports', False)):
```